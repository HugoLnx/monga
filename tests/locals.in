void main() {
	int num;
	char c;
	int num2;
	num = 99;
	c = 'D';
	num2 = 18;
	printf("num: %d\n", num);
	printf("c: %c\n", c);
	printf("num2: %d", num2);
}
