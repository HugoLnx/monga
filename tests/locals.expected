num: 99
c: D
num2: 18
